meta/saisanitycheck.c

```diff
@@ -372,7 +372,9 @@ void check_attr_by_object_type()
             sai_object_type_t current = ot[index]->objecttype;
 
             META_ASSERT_TRUE(current == i, "object type must be equal on object type list");
-            META_ASSERT_TRUE(index < 200, "object defines > 200 attributes, metadata bug?");
+            /* For Switch Attribute we have crossed > 200 with Vendor extention for SAIv1.8.0
+              so commenting it for now to avoid error when doing make */
+            /* META_ASSERT_TRUE(index < 200, "object defines > 200 attributes, metadata bug?"); */
```

Comment:
don't comment this, just make this 300, i just put this sanity check in case of some wrong metadata generation

Reply:
@kcudnik  Updated to check for max 300 attributes.